src/components/production/ProductionTile.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div class="px-4 cursor-pointer production-tile sm:px-0">
+    <router-link
+      :to="{ name: 'production', params: { productionSlug: production.slug } }"
+    >
+      <img
+        class="max-w-full"
+        src="https://via.placeholder.com/1920x960"
+        alt="show poster"
+      />
+      <h2
+        class="text-2xl font-semibold transition-color text-sta-orange production-title"
+      >
+        {{ production.name }}
+      </h2>
+      <h2 class="text-2xl transition-colors">
+        {{ displayStartEnd(production.start, production.end, 'd MMM') }}
+      </h2>
+    </router-link>
+  </div>
+</template>
+
+<script>
+import { displayStartEnd } from '@/utils';
+export default {
+  props: {
+    production: {
+      required: true,
+      type: Object,
+    },
+  },
+  methods: {
+    displayStartEnd,
+  },
+};
+</script>
+
+<style scoped lang="scss">
+.production-tile:hover {
```

Comment:
can this not be tailwind?

Reply:
Unfortunately not - we want the test to change colour when the user hovers over any part of the tile. We could do this with tailwind if the show name heading didn't have a manual colour assigned, but this takes priority over the hover colour so you have to do it with css.

Reply:
i seeeee